test/cpp/end2end/xds/xds_end2end_test_lib.h

```diff
@@ -720,7 +721,7 @@ class XdsEnd2endTest : public ::testing::TestWithParam<XdsTestType> {
     // Dev note: If a timeout or Deadline Exceeded error is occurring in an XDS
     // end2end test, consider changing that test's timeout instead of this
     // global default.
-    int timeout_ms = 1000;
+    int timeout_ms = BuiltUnderMsan() ? 2000 : 1000;
```

Comment:
should this be `grpc_test_slowdown_factor()*1000`?

Reply:
We can, but it's more of a shotgun approach. That'd be setting expected slowdowns of:
* 4x for MSAN
* 5x for UBSAN (which is not a problem for XDS tests)
* 3x for ASAN (which is not a problem for XDS tests)
* 5x for TSAN (which is not a problem for XDS tests)

Reply:
The test framework uses `grpc_timeout_milliseconds_to_deadline()` on the value of `timeout_ms`:

https://github.com/grpc/grpc/blob/03b6b01043f778e76b1f1380f5619067a7c5a42a/test/cpp/end2end/xds/xds_end2end_test_lib.cc#L419

And `grpc_timeout_milliseconds_to_deadline()` already multiplies by `grpc_test_slowdown_factor()`:

https://github.com/grpc/grpc/blob/93fbacb5343ea8cf993f779cee0847aed107bdc7/test/core/util/test_config.cc#L90

If we multiplied by that factor here, we'd be applying that multiplier twice, which I don't think we want.